src/debugger/DebuggerSha1.cpp

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (c) 2016-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+/*
```

Comment:
As I know, we cannot embed apache2 source code. both licences are conflicting

Reply:
I need some help here. All three scripts here has this license, and I just copied them into the debugger code.
https://github.com/Samsung/escargot/tree/master/tools

Shall I copy the license from the C files?

Reply:
And one more question, is it ok to put the python debugger client into tools/debugger or do you prefer some other directory?

Reply:
Directory is not problem.
We cannot link lgpl2 with apache2 into same exe file(or so).
tools can be apache2. that files are not linked into our binary
Can you replace or remove apache2 source codes from this file?

Reply:
I am sorry, I confused this code with the python files (which used Apache).

Reply:
I cannot remove this license since the original code was coming from an embedded OS, but we can replace this with another Sha1 implementation (its a standard after all). I will try to find another, but if you recommend one, we could just use that. Unfortunately I don't know enough about Sha1 coding to create one myself.